widgets/common/src/InputsUtils.tsx

```diff
@@ -366,16 +366,24 @@ class InputsUtils {
         }
     }
 
+    static evaluateValue(input, inputsState, dataType) {
+        if (input.type === 'integer' && Number.isNaN(inputsState[input.name])) {
```

Comment:
Shouldn't we also take care about floating numbers?
```suggestion
        if ((input.type === 'integer' || input.type === 'float') && Number.isNaN(inputsState[input.name])) {
```

Ref.: See possible input types in https://docs.cloudify.co/latest/developer/blueprints/spec-inputs/ - Schema -> Type